public/give.html

```diff
@@ -174,7 +228,16 @@
 <main class="content">
     <div class="container" style="padding-bottom: 40px;">
       <div class="map-container" style="display: none;">
-        <h2 style="padding: 30px 0px 20px 0px;" class='i18n' data-i18n="ftm-map-of-sites">Map of donation sites</h2>
+        <h2 style="padding-top: 30px;" class='i18n' data-i18n="ftm-map-of-sites">Map of donation sites</h2>
+
+        <div class="map-search-wrap">
+          <input type="search" name="map-search" id="map-search" placeholder="Enter ZIP code or address, e.g. 94070">
```

Comment:
Needs `i18n`.

Reply:
@raindrift do you know if it's possible to apply i18n to attributes? Not totally familiar with the module.